src/lib/utils/did.test.ts

```diff
@@ -0,0 +1,153 @@
+import { isAbbreviatedVerkey, isDid, isDidIdentifier, isFullVerkey, isVerkey } from './did';
+
+const validAbbreviatedVerkeys = [
+  '~PKAYz8Ev4yoQgr2LaMAWFx',
+  '~Soy1augaQrQYtNZRRHsikB',
+  '~BUF7uxYTxZ6qYdZ4G9e1Gi',
+  '~DbZ4gkBqhFRVsT5P7BJqyZ',
+  '~4zmNTdG78iYyMAQdEQLrf8',
+];
+
+const validFullVerkeys = [
+  '6YnVN5Qdb6mqimTRQcQmSXrHXKdTEdRn5YHZReezUTvt',
+  '8jG2Bim1HNSybCTdKBRppP4PCQSSijx1pBnreqsdo8JG',
+  '9wMLhw9SSxtTUyosrndMbvWY4TtDbVvRnMtzG2NysniP',
+  '6m2XT39vivJ7tLSxNPM8siMnhYCZcdMxbkTcJDSzAQTu',
+  'CAgL85iEecPNQMmxQ1hgbqczwq7SAerQ8RbWTRtC7SoK',
+];
+
+const validDids = [
+  'did:indy:BBPoJqRKatdcfLEAFL7exC',
+  'did:sov:N8NQHLtCKfPmWMgCSdfa7h',
+  'did:random:FBSegXg6AsF8J73kx22gjk',
+  'did:sov:8u2b8ZH6sHeWfvphyQuHCL',
+  'did:ethr:0xb9c5714089478a327f09197987f16f9e5d936e8a',
+  'did:btcr:xyv2-xzpq-q9wa-p7t',
+];
+
+const validDidIdentifiers = [
+  '8kyt-fzzq-qpqq-ljsc-5l',
+  '9noxi4nL4SiJAsFcMLp2U4',
+  'QdAJFDpbVoHYrUpNAMe3An',
+  'B9Y3e8PUKrM1ShumWU36xW',
+  '0xf3beac30c498d9e26865f34fcaa57dbb935b0d74',
+];
+
+describe('Utils | Did', () => {
+  describe('isAbbreviatedVerkey()', () => {
+    it('should return true when passed in verkey is an abbreviated verkey', () => {
```

Comment:
Sure, this is just my personal convention for writing tests. I'll change it. But I'll like to make it `returns` instead of `return` then. Is that okay?

```
 PASS  src/lib/utils/did.ts (17.982 s)
  isAbbreviatedVerkey()
    ✓ returns true when passed in verkey is an abbreviated verkey (3332 ms)
```

When running tests, it will read like a story when you combine the `describe` and `test`:

```
isAbbreviatedVerkey() returns true when passed in verkey is an abbreviated verkey
```

That's why I normally use `should`:

```
isAbbreviatedVerkey() should return true when passed in verkey is an abbreviated verkey
```

Reply:
Yeah, `returns` is good.